trade-finance-manager-ui/server/types/view-models/deal-cancellation/effective-from-date-view-model.ts

```diff
@@ -6,6 +6,11 @@ export type EffectiveFromDateErrorsViewModel = {
   effectiveFromDateError: { message: string; fields: string[] };
 };
 
+export type EffectiveFromDateValidationViewModel = {
+  errors: EffectiveFromDateErrorsViewModel | null;
```

Comment:
I think this can just be `errors?: EffectiveFromDateErrorsViewModel` ?

Not a blocker as seems we could do this in a few places